lib/actions/upload.js

```diff
@@ -1,79 +1,47 @@
-'use strict';
-const Client = require('ssh2-sftp-client');
-const request = require('request');
+/* eslint-disable no-await-in-loop */
+const { Logger } = require('@elastic.io/component-commons-library');
+const { promisify } = require('util');
+const request = promisify(require('request'));
```

Comment:
Better to upload attachments with `component-commons-library` imstead of `request`